python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -442,3 +442,92 @@ def hybrid_forward(self, F, x):
 
     def __repr__(self):
         return self.__class__.__name__
+
+
+class InstanceNorm(HybridBlock):
+    """
```

Comment:
missing argument documentation

Reply:
Thanks! The args are added here: https://github.com/apache/incubator-mxnet/pull/7570/files#diff-06285baf97b6456dd23bdb99f6e239b6R461